pygam/pygam.py

```diff
@@ -2357,7 +2384,7 @@ def gridsearch(self, X, y, weights=None, return_scores=False,
         if progress:
             pbar = ProgressBar()
         else:
-            pbar = lambda x: x
+            def pbar(x): return x
```

Comment:
@maorn 
whats the intuition behind this change?

Reply:
ahh for pickling?

Reply:
my eclipse autocorrects any pep8 and pylint errors and warnings. this is one of these changes (i didn't change it manually) 
basic explanation for this change (pep8 E731)
https://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def